src/routes/InstancesPage/InstancesPage.js

```diff
@@ -279,13 +279,14 @@ function InstancesPage() {
                     key={instance.name}
                     onRowClick={(event) => {
                       if (
-                        event.target.getAttribute('type') !== 'button' &&
```

Comment:
this was causing inconsistent clicking behavior if the user was clicking on the SVG of the dropdown (kebab) instead of the button